maintainers/maintainer-list.nix

```diff
@@ -11544,6 +11544,12 @@
     githubId = 6740669;
     name = "Tom Smeets";
   };
+  tonyshkurenko = {
```

Comment:
Maybe for later and future PR, I think that we should have a twingate github user for those purposes

Reply:
They've asked, I've added, don't know about this, I checked a couple of packages, all of them have same structure

Reply:
They?